chacha20poly1305/CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.9.0 (2021-08-29)
+### Changed
+- Bump `chacha20` to v0.9: now a hard dependency ([#365])
+- MSRV 1.51+ ([#365])
+
+### Removed
+- `chacha20` feature: now a hard dependency ([#365])
+- `xchacha20` feature: now always-on by default ([#365])
```

Comment:
"By default" implies it could be turned off, which I believe is not the case? It's just always-present.

Reply:
Changed it to just say "now always-on"